hris_publisher/hris.py

```diff
@@ -11,15 +11,15 @@
 
 
 sl = utils.StructuredLogger(name=__name__, level=logging.DEBUG)
-logger = logging.getLogger(__name__)
+logger = logging.getLogger('cis_hris')
 
 
 class HrisJSON(object):
     """Retrieval and serialization object for the current HRIS data file."""
     def __init__(self, boto_session):
         self.boto_session = boto_session
         self.from_file = False
-        self.file_name = 'hris.json'
+        self.file_name = 'workday.json'
```

Comment:
i think this breaks the test, maybe we can copy the file name or change the test if required